baseline/OSPS-BR.yaml

```diff
@@ -451,6 +472,10 @@ controls:
           - reference-id: SA-15
           - reference-id: SI-7
           - reference-id: SI-7(14)
+      - reference-id: UKSSCOP
+        entries:
+          - reference-id: Claim 1.2.2
```

Comment:
This control is more about output than input. 1.2.2 is about input, and 3.1.1 is output, so we should drop 1.2.2 here imo.

Reply:
I just noticed that we have an entry for "UKSSCOP" to 3.1 on line 446.

In fact, we already seem to have mappings for UKSSCOP which disagree with the more fine-grained ones we are making now.